Sources/PointFree/Episode/Episode.swift

```diff
@@ -5,44 +5,59 @@ public struct Episode {
   public private(set) var blurb: String
   public private(set) var codeSampleDirectory: String
   public private(set) var exercises: [Exercise]
+  public private(set) var fullVideo: Video?
   public private(set) var id: Id
   public private(set) var image: String
+  public private(set) var itunesImage: String?
```

Comment:
iTunes needs a square version of all of our poster images, so need to upload that assets in addition to the posters we use on the site.